Badge/badge.css

```diff
@@ -0,0 +1,49 @@
+
+.badge-small-size
+{
+    height: 12px;
+    width: 12px;   
+    border-radius: 50%;
+    display: inline-block;
+    position:relative;
+    right:0.75em;
+  }
+
+  .badge-normal-size{
+    height: 18px;
+    width: 18px;
+    
+    border-radius: 50%;
+    display: inline-block;
+    position:relative;
+    right:1.5em;
+  }
+
+  .badge-large-size{
+    height: 25px;
+    width: 25px;
+   
+    border-radius: 50%;
+    display: inline-block;
+    position:relative;
+    right:1.95em;
+  }
+
+.badge-on-icon{
+    height:12px;
+    width:12px;
+    border-radius: 50%;
+    display: inline-block;
+    position:relative;
+    bottom:2em;
+    right:0.75em;
+    font-size: 12px;
+    text-align: center;
+}
+.badge-within{
+    border-radius: 50%;    
+    margin-left:1em;
+    height:25px;
+    width:25px;
+    
```

Comment:
here if you give a padding as below: 
padding: .75em 1em;
the badge will look good.